packages/manager/modules/pci/src/projects/project/kubernetes/onboarding/translations/Messages_en_GB.json

```diff
@@ -1,21 +1,21 @@
 {
- "pci_projects_project_kubernetes_title": "Kubernetes",
- "pci_projects_project_kubernetes_onboarding_content1": "You have not yet created a Managed Kubernetes service.",
- "pci_projects_project_kubernetes_onboarding_content2": "Orchestrate your containerized applications with a CNCF-certified Kubernetes cluster.",
- "pci_projects_project_kubernetes_onboarding_content3": "With Managed Kubernetes, start one or more Kubernetes clusters to orchestrate your containerized applications in the OVH cloud. The master service is provided free of charge, only instances and volumes used by your worker nodes will be charged at the normal price.",
- "pci_projects_project_kubernetes_onboarding_action_label": "Create a cluster",
- "pci_projects_project_kubernetes_onboarding_guides_configure_title": "Configuring kubectl on an OVH Managed Kubernetes cluster",
- "pci_projects_project_kubernetes_onboarding_guides_configure_description": "",
- "pci_projects_project_kubernetes_onboarding_guides_nodes_title": "Adding nodes to a OVH Managed Kubernetcluster",
- "pci_projects_project_kubernetes_onboarding_guides_nodes_description": "",
- "pci_projects_project_kubernetes_onboarding_guides_deploy-app_title": "Deploying an application",
- "pci_projects_project_kubernetes_onboarding_guides_deploy-app_description": "",
- "pci_projects_project_kubernetes_onboarding_guides_load-balancer_title": "Using the OVH Managed Kubernetes LoadBalancer",
- "pci_projects_project_kubernetes_onboarding_guides_load-balancer_description": "",
- "pci_projects_project_kubernetes_onboarding_guides_helm_title": "Installing Helm on OVH Managed Kubernetes",
- "pci_projects_project_kubernetes_onboarding_guides_helm_description": "",
- "pci_projects_project_kubernetes_onboarding_guides_istio_title": "Installing Istio on OVH Managed Kubernetes",
- "pci_projects_project_kubernetes_onboarding_guides_istio_description": "",
- "pci_projects_project_kubernetes_onboarding_regions_unavailable": "Your current Public Cloud project is configured in a region where Kubernetes is not available, it is necessary to create a new project in order to deploy a Kubernetes cluster, we are working to extend the Kubernetes offer to other regions.",
- "pci_projects_project_kubernetes_onboarding_regions_create": "Create a new public Cloud project."
-}
\ No newline at end of file
+  "pci_projects_project_kubernetes_title": "Kubernetes",
```

Comment:
could you please revert changes on this file if it's just formating

it will allow us to not having an unecessary change on pci module and thus avoiding the CI to detect a change on public-cloud app